packages/storybook/stories/snippets/ErrorHandlingStatefulClient.snippet.tsx

```diff
@@ -0,0 +1,102 @@
+import { Call, CallAgent } from '@azure/communication-calling';
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+import {
+  FluentThemeProvider,
+  DEFAULT_COMPONENT_ICONS,
+  CallClientProvider,
+  CallAgentProvider,
+  CallProvider,
+  createStatefulCallClient,
+  StatefulCallClient,
+  CallClientState
+} from '@azure/communication-react';
+import { initializeIcons, registerIcons } from '@fluentui/react';
+import React, { useEffect, useState } from 'react';
+
+initializeIcons();
+registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
+
+export interface CallScreenProps {
+  userAccessToken: string;
+  userId: string;
+  groupId: string;
+  displayName: string;
+}
+
+export const CallScreen = (props: CallScreenProps): JSX.Element => {
+  const { userAccessToken, userId, groupId, displayName } = props;
+
+  const [statefulCallClient, setStatefulCallClient] = useState<StatefulCallClient>();
+  const [callAgent, setCallAgent] = useState<CallAgent>();
+  const [call, setCall] = useState<Call>();
+
+  /**
+   * This state we will store the message from the latest error that was emitted.
+   */
+  const [latestError, setLatestError] = useState<string>();
```

Comment:
Should we use the latestError in the CallScreen component?

Reply:
Nah this is just an example for what they can do. figured kicking it out to state would be good enough